### ISSUE TITLE
Transparent hair fill with only the hairStyle outline present

### ISSUE THREAD

**Issue by WilliamRADFunk** [dicebear/dicebear#129]
<img width="61" alt="transparent-hair-example-01" src="https://user-images.githubusercontent.com/10148684/122973297-ef8afe80-d345-11eb-9ac4-c2ce4a5247fb.PNG">

<img width="57" alt="transparent-hair-example-02" src="https://user-images.githubusercontent.com/10148684/122973312-f4e84900-d345-11eb-867c-1be9ba15e2c8.PNG">

These are the values I'm passing in during the createAvatar use:

![image](https://user-images.githubusercontent.com/10148684/122974157-db93cc80-d346-11eb-94bd-af930413c2f5.png)

The vast majority of the avatars work fine. Even those hair styles work in other avatars. No errors or warnings in the console, either.

Consistent code example:

`return createAvatar(
    style,
    {
        backgroundColor: 'salmon',
        seed: 'Mic-Earthworm-Hazem',
        facialHairColor: ['topaz'],
        facialHairProbability: 0,
        glassesProbability: 0,
        hair: ['fonze'] as Hair,
        hairColor: ['topaz'],
        mouth: ['smirk']
    });`

**Comment by FlorianKoerner** [comment]
Hi @WilliamRADFunk 

Can you please share your sample code as text?

**Comment by WilliamRADFunk** [comment]
Sure. I've also narrowed it down to a consistent set of options to make it repro every time:

`return createAvatar(
    style,
    {
        backgroundColor: 'salmon',
        seed: 'Mic-Earthworm-Hazem',
        facialHairColor: ['topaz'],
        facialHairProbability: 0,
        glassesProbability: 0,
        hair: ['fonze'] as Hair,
        hairColor: ['topaz'],
        mouth: ['smirk']
    });`

**Comment by FlorianKoerner** [comment]
I can't reproduce the error with your first code example.

<img width="300" alt="Bildschirmfoto 2021-06-22 um 20 25 46" src="https://user-images.githubusercontent.com/647303/122981630-40c4dd80-d39a-11eb-9401-ff63efb816ba.png">

With the second one I can reproduce it. It was due to a logic that says that the hair must not have the same color as the face. And because only one color was given as an option, there was no alternate color.

I have published a new release. As soon as only one color is specified, this color is now also enforced.

https://github.com/dicebear/dicebear/releases/tag/v4.6.5
